PLUB/Sources/Views/Home/Cell/Home/RecommendedMeetingCollectionViewCell.swift

```diff
@@ -21,28 +21,29 @@ class RecommendedMeetingCollectionViewCell: UICollectionViewCell {
   private let registerInterestView = RegisterInterestView().then {
     $0.layer.masksToBounds = true
     $0.layer.cornerRadius = 20
-    $0.backgroundColor = .secondarySystemBackground
+    $0.backgroundColor = .lightGray
     $0.isUserInteractionEnabled = true
   }
   
   class RegisterInterestView: UIView {
     
     private let label = UILabel().then {
       $0.text = "관심사 등록 하기"
-      $0.textColor = .gray
-      $0.font = .systemFont(ofSize: 25, weight: .bold)
+      $0.textColor = .deepGray
+      $0.font = .h5
     }
     
     private let imageView = UIImageView().then {
       let config = UIImage.SymbolConfiguration(pointSize: 40, weight: .bold)
       let image = UIImage(systemName: "plus.circle", withConfiguration: config)
       $0.image = image
       $0.contentMode = .scaleAspectFill
-      $0.tintColor = .gray
+      $0.tintColor = .deepGray
     }
     
     override init(frame: CGRect) {
       super.init(frame: frame)
+      backgroundColor = .lightGray
       _ = [label, imageView].map{ addSubview($0) }
```

Comment:
`map` 함수는 어떠한 값을 다른 유형의 값으로 `매핑(mapping)`하는 성격이 강합니다.
해당 구문에서는 값을 할당하는 게 아니라 for문과 같이 리스트 요소를 반복적으로 꺼내어 작업하는 것처럼 보이므로 `forEach`구문이 적절하지 않나 생각해봅니다.
```suggestion
      [label, imageView].forEach { addSubview($0) }
```

Reply:
음 사실 이 부분은 이전 개발자분께서 조언해주셔서 map으로 했던 부분인데요 !! foreach로 변경해도 무관합니다 ✔️✔️ 
수정하도록 하겠습니다 😁😁

Reply:
어느 분께서 어떻게 조언해주셨는지 말씀해주실 수 있나요? 그 분의 의견이 궁금하네요. 링크가 있다면 혹시 올려주실 수 있나요?